src/System.Net.WebProxy/tests/WebProxyTest.cs

```diff
@@ -145,7 +145,7 @@ public static IEnumerable<object[]> BypassOnLocal_MemberData()
             yield return new object[] { new Uri($"http://{Guid.NewGuid().ToString("N")}"), true };
             foreach (IPAddress address in Dns.GetHostEntryAsync(Dns.GetHostName()).GetAwaiter().GetResult().AddressList)
             {
-                if (address.AddressFamily == AddressFamily.InterNetwork)
+                if (address.AddressFamily == AddressFamily.InterNetwork && !IPAddress.IsLoopback(address))
```

Comment:
No, this is not a test issue since it doesn't happen on anything but Ubuntu when DHCP is configured.
More likely a GetHostEntryAsync behavior difference on *NIX.